Sources/Loaf/LoafView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  LoafView.swift
+//  Loaf
+//
+//  Created by Joseph Roque on 2020-01-16.
+//  Copyright © 2020 Mat Schmid. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+public extension View {
+    /// Presents a Loaf.
+    ///
+    /// - Parameters:
+    ///   - loaf: A binding to a Loaf. If the loaf changes, the system will dismiss the currently-presented Loaf
+    ///     and replace it with a new Loaf.
+    ///   - onDismiss: called when the Loaf times out or is tapped
+    func loaf(_ loaf: Binding<Loaf?>, onDismiss: Loaf.LoafCompletionHandler = nil) -> some View {
+        ZStack {
+            LoafView(loaf, onDismiss: onDismiss)
+            self
+        }
+    }
+}
+
+@available(iOS 13.0, *)
+struct LoafView: UIViewControllerRepresentable {
+    @Binding var loaf: Loaf?
+    private let onDismiss: Loaf.LoafCompletionHandler
+    
+    init(_ loaf: Binding<Loaf?>, onDismiss: Loaf.LoafCompletionHandler) {
+        _loaf = loaf
+        self.onDismiss = onDismiss
+    }
+    
+    func makeUIViewController(context: Context) -> LoafWrappedViewController {
+        return LoafWrappedViewController(loaf: loaf)
+    }
+    
+    func updateUIViewController(_ uiViewController: LoafWrappedViewController, context: Context) {
+        if let loaf = self.loaf {
+            if let presentedLoaf = uiViewController.loaf, presentedLoaf.message != loaf.message {
```

Comment:
Here we assume that two loafs with different messages are wholly different loafs. Might be a bad assumption to make, but we need a way to differentiate loafs being passed to the function. In practice, this assumption probably isn't a big deal.

By making this assumption, we can immediately dismiss the current loaf and present a new loaf when a new loaf is passed to the function. If the new loaf has the same message, the old loaf will have to finish presenting before the new one is displayed.